src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -53,7 +64,8 @@ public ModelManager() {
         this(new AddressBook(), new Wallet(), new UserPrefs());
     }
 
-    // =========== UserPrefs ==================================================================================
+    // =========== UserPrefs
+    // ==================================================================================
```

Comment:
why u edited this haha

Reply:
Oops, I think my Intellij a little messed up, will update this!